main/ui/src/main/java/org/cryptomator/ui/preferences/GeneralPreferencesController.java

```diff
@@ -109,7 +121,11 @@ public void toggleAutoStart() {
 		autoStartStrategy.ifPresent(autoStart -> {
 			boolean enableAutoStart = autoStartCheckbox.isSelected();
 			Task<Void> toggleTask = new ToggleAutoStartTask(autoStart, enableAutoStart);
-			toggleTask.setOnFailed(evt -> autoStartCheckbox.setSelected(!enableAutoStart)); // restore previous state
+			toggleTask.setOnFailed(event -> {
+				autoStartCheckbox.setSelected(!enableAutoStart); // restore previous state
+				LOG.error("Failed to toggle autostart.", event.getSource().getException());
+				errorComponent.cause(event.getSource().getException()).window(window).returnToScene(preferencesScene.get()).build().showErrorScene();
+			});
```

Comment:
An alternative would be to get this scene over the window: `window.getScene()` The benefit would be in reducing code (variable definitions and object injections)

Reply:
Resolved by d331c3022489c7d0651a588e0c58f877064d55a8.